vendor/modules.txt

```diff
@@ -1,3 +1,6 @@
+# github.com/Jeffail/gabs/v2 v2.6.1
+## explicit
+github.com/Jeffail/gabs/v2
```

Comment:
Should be in the first and separated commit as mentioned in the first commit's comment.